styles.css

```diff
@@ -296,3 +296,112 @@
 .project-button:hover {
   box-shadow: 1px 8px 16px rgba(125, 115, 252, 0.48);
 }
+
+.contact-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.contact-container h3 {
+  width: 327px;
+  height: 35px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #9390ff;
+  margin-bottom: 24px;
+  margin-top: 120px;
+}
+
+.contact-container p {
+  width: 327px;
+  height: 105px;
+  font-family: Inter, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  color: #f5f5ff;
+  margin-bottom: 40px;
+}
+
+.form-input-container {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.input {
+  width: 327px;
+  height: 48px;
+  background-color: #fff;
+  margin: 40px 24px;
+  padding: 15px 16px;
+}
+
+.client-message {
+  width: 327px;
+  height: 148px;
+  background-color: #fff;
+  font-family: Inter, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #81818c;
+  padding: 12px 12px;
+}
```

Comment:
- [ ] It will look closer to the Figma template if you will fix the margins. In your contact form, the margins are too much. 
> Your form:
<img width="275" alt="Screen Shot 2022-01-12 at 02 00 27" src="https://user-images.githubusercontent.com/49064106/149028704-bc42685b-6fd5-485d-a0b7-19887058d9a9.png">

> Figma template:
<img width="207" alt="Screen Shot 2022-01-12 at 02 04 40" src="https://user-images.githubusercontent.com/49064106/149028759-8bc0e9b2-43c9-48e8-9e34-938df6aff8bd.png">